source/publish/lizmap_plugin/editing.rst

```diff
@@ -155,3 +155,65 @@ This allow to store the sent media files and images in a centralized folder, for
   - media
 
     * tree_pictures/ <-- to store images in this folder, use: :file:`media/tree_pictures/`
+
+Use a WebDAV remote storage
+_________________________________
+
+It is possible to store your files in a remote server that supports the `WebDAV protocol` (e.g. Nextcloud). To achieve this you must:
+
+- Configure the :guilabel:`Attachments` edit widget for the choosen field as follow:
+
+  1. In the :guilabel:`Storage type` field, select the `WebDAV Storage` option
+  2. In the :guilabel:`External Storage`` section, on the :guilabel:`Store URL` field, add an :code:`expression` indicates the full url of the remote path for the file. The expression string must:
+
+    * start with the remote storage url of the :code:`root` folder of the WebDAV server
+    * include or not any destination subfolder 
+    * end with the :code:`file_name(@selected_file_path)` expression
+
+  3. update the :guilabel:`Authentication` section with your :code:`user name` and :code:`password` (optional)
```

Comment:
> optional, read below about authentication

or something similar ? Because later we read we must set the same value as in this widget.